docs/addons/karpenter.md

```diff
@@ -10,17 +10,15 @@ Karpenter works by:
 * Scheduling the pods to run on the new nodes, and
 * Removing the nodes when the nodes are no longer needed
 
-***IMPORTANT***:
+## Prerequisites
 
 1. This add-on depends on [VPC CNI](vpc-cni.md) Add-on for cni support.
 
 ***VPC CNI add-on must be present in add-on array*** and ***must be in add-on array before the Karpenter add-on*** for it to work, as shown in below example. Otherwise will run into error `Assertion failed: Missing a dependency for VpcCniAddOn`.
 
 2. There is no support for utilizing both Cluster Autoscaler **and** Karpenter. Therefore, any addons list that has both will result in an error `Deploying <name of your stack> failed due to conflicting add-on: ClusterAutoscalerAddOn.`.
 
-## Prerequisite
-
-(If using Spot), EC2 Spot Service Linked Role should be created. See [here](https://docs.aws.amazon.com/batch/latest/userguide/spot_fleet_IAM_role.html) for more details.
+3. (If using Spot), EC2 Spot Service Linked Role should be created. See [here](https://docs.aws.amazon.com/batch/latest/userguide/spot_fleet_IAM_role.html) for more details.
```

Comment:
Do you think customers will benefit from doing this automatically with blueprints? I understand the lifecycle for this is account, rather than a cluster, interested in your thoughts on it. If a pattern in the patterns repo with spot and karpenter is useful, let's create an issue for it so that one of us could address.

Reply:
I can't put a number around it - but I saw a conversation where around half of the the spot customers on EKS consider Karpenter, so generally it's a pattern we should consider. I'm happy to create an issue on the patterns repo and include this check - this might be revised as a troubleshooting note on this repo (i.e. if you select spot, you may run into errors if you don't do this)

Reply:
I created the issue on the patterns repo.